netplan/cli/commands/try_command.py

```diff
@@ -47,13 +47,24 @@ def __init__(self):
         self._config_manager = None
         self.t_settings = None
         self.t = None
+        self._rootdir = '/'
+        if 'DBUS_TEST_NETPLAN_ROOT' in os.environ:
+            self._rootdir = os.environ['DBUS_TEST_NETPLAN_ROOT']
+        self._netplan_try_stamp = os.path.join(self._rootdir, 'run', 'netplan-try.ready')
 
     @property
     def config_manager(self):  # pragma: nocover (called by later commands)
         if not self._config_manager:
             self._config_manager = ConfigManager()
         return self._config_manager
 
+    def clear_ready_stamp(self):
+        if os.path.isfile(self._netplan_try_stamp):
```

Comment:
(ntpick^3) this is a potential TOCTOU issue, probably utterly irrelevant here, it could be a problem in a multi-threaded environment when clear_ready_stamp is called in two threads - which is not really an issue, just noticed it while reviewing and couldn't help but mention it.

Reply:
`netplan try` is not designed to run two instances or threads in parallel, if we'd want to enable this, we'd probably need a much bigger refactoring. So I'll leave this as is for now.